localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -69,47 +69,47 @@
  innovation_naga_taming_desc:0 "We have tamed the Couatl and enslaved the Sea Giants, and now we have encorporated them into our army."
  
  innovation_savage:0 "Savages"
- innovation_savage_desc:0 ""
+ innovation_savage_desc:0 "Fanatic and barbaric warriors of the Bleeding Hollow clan, savages are strong and subborn, and show no mercy."
  
  innovation_silithid:0 "Silithid"
  innovation_silithid_desc:0 "Silithids are insectoid creatures that are very strong and sophisticated. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. It seems that the Silithids' intention is to devour everything that stands in their way and that does not belong to their own ecology. This is the only thing they focus on, and this is why they pose a great threat to all life on Azeroth."
 
  innovation_snow_hunter:0 "Snow Hunters"
- innovation_snow_hunter_desc:0 ""
+ innovation_snow_hunter_desc:0 "Our kind has had to bend the elements to our will in order to survive.  The best of our warriors are the hunters who keep us safe."
     
  innovation_thornguard:0 "Thornguards"
  innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders.  They are heavily armed and armored; they will stop any who dare to trespass or any who dare to not submit."
 
  innovation_tidehunter:0 "Tidehunters"
- innovation_tidehunter_desc:0 ""
+ innovation_tidehunter_desc:0 "mrglmrglmrglmrgl"
  
  innovation_transcendent:0 "Transcendent Avatars"
  innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: Spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
     
  innovation_ursa:0 "Ursas"
- innovation_ursa_desc:0 ""
+ innovation_ursa_desc:0 "Ursas are the biggest, meanest and most feral of furbolgs. They tower above humans and orcs, and can snap elven necks with single paw swipes. They are immensely strong and tough, and focus on improving their natural, bearlike natures. They disembowel dragons with their claws and crush small creatures that come within their grasp."
```

Comment:
Why are they mean? Aren't they just the strongest champions of the furbolg? Doesn't have to mean that they're bad.
And disemboweling dragons? Seems a little too strong imho.

Reply:
Just copied this, like mostly all of the loc from wiki, if you think its too much ill change